setup.cfg

```diff
@@ -15,3 +15,12 @@ exclude = *build/,info.py,_version.py,due.py
 putty-ignore =
     */__init__.py : +F401
 ignore = E203,E402,E722,W503
+
+[options.extras_require]
```

Comment:
you can drop this section. `setup.cfg` is my enemy now.

Reply:
Done